algorithms/loopclosure/loop-closure-handler/test/test_loop_closure_handling_test.cc

```diff
@@ -407,6 +298,18 @@ void LoopClosureHandlerTest::addDuplicateLandmarksToQueryKeyframes() {
       constraints_.push_back(constraint);
     }
   }
+
+  for (unsigned int j = kNumOfDuplicateLandmarks; j < kNumOfLandmarks; j++) {
```

Comment:
To avoid the implicit copy:
```suggestion
  for (unsigned int j = kNumOfDuplicateLandmarks; j < kNumOfLandmarks; ++j) {
```